polar-core/benches/benchmarks/queries.rs

```diff
@@ -132,10 +132,11 @@ pub fn prime(c: &mut Criterion) {
 pub fn indexed_rules(c: &mut Criterion) {
```

Comment:
Can someone verify that we're still benchmarking the right thing after these changes? I think we are.

Reply:
Yep